main.py

```diff
@@ -1,40 +1,41 @@
 import os, sys
 from PIL import Image
-import functions as Fn
+from functions import ImageResizer
 
-def menu():
-    os.system('cls')
-    print '''
-    =====================
-    Image Editor
-    =====================
-    1 - Cut Image
-    2 - Change Format
-    3 - Make thumbnail
-    4 - Close
-    =====================
-    '''
-    choice = raw_input('Choose and press Enter: ')
-    return choice
+class Main:
+    def menu(self):
+        os.system('cls')
+        print '''
+        =====================
+        Image Editor
+        =====================
+        1 - Cut Image
+        2 - Change Format
+        3 - Make thumbnail
+        4 - Close
+        =====================
+        '''
+        choice = raw_input('Choose and press Enter: ')
+        return choice
 
+m = Main()
+IR = ImageResizer()
 # Choose function
 choice = ""
 while choice != "4":
```

Comment:
I assume that this loop should be in some function in the class.. otherwise this does not have anything to do with objects.